FlashCards.ZubePierreBasali/Database/DataTools.cs

```diff
@@ -0,0 +1,240 @@
+﻿using FlashCards.FlashCardsManager.Models;
+using Spectre.Console;
+using System.Configuration;
+using Dapper;
+using System.Data;
+using System.Data.SqlClient;
+
+using FlashCards.StudySessions;
+
+namespace FlashCards.Database
+{
+    internal class DataTools
+    {
+        internal string ConnectionString { get; set;}
+
+        internal void Initialization()
+        {
+            try
+            {
+                string sqlCommandText;
+                Configuration config = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+                try
+                {
+                    if (config.AppSettings.Settings["connectionString"].Value == "")
+                    {
+                        string connectionString = @$"Data Source=localhost;Integrated Security=SSPI;Initial Catalog=;TrustServerCertificate=True;";
```

Comment:
🟠 **Redundant**

💡 This only works on your environment. Better to throw an exception if missing.